_playwright-tests/style_guide.md

```diff
@@ -0,0 +1,238 @@
+# Playwright Style Guide 💅
+
+This style guide outlines the standards and best practices for writing Playwright tests based on the experience gathered during the creation of the Playwright PoC and further efforts.
+
+Most of the things here are in-line with the [official Playwright best practices/guidelines](https://playwright.dev/docs/best-practices).
+
+> [!TIP]
+> The official Playwright documentation is **awesome** and should be your first stop for any questions you have about Playwright. Often this will pay of more than jumping into AI chat-bots first.
```

Comment:
```suggestion
> The official Playwright documentation is **awesome** and should be your first stop for any questions you have about Playwright. Often this will pay off more than jumping into AI chat-bots first.
```